packages/fast-css-editor-react/src/position/index.tsx

```diff
@@ -0,0 +1,173 @@
+                value={this.props[location] || ""}
+            />
+        )
+    }
+
+    private generateCenterRowClassNames(): string {
+        let classNames: string = this.props.managedClasses.absoluteInput_row_center;
+
+        [
+            {
+                location: Location.top,
+                className: this.props.managedClasses.absoluteInput_row_center__activeTop
+            },
+            {
+                location: Location.bottom,
+                className: this.props.managedClasses.absoluteInput_row_center__activeBottom
+            },
+            {
+                location: Location.left,
+                className: this.props.managedClasses.absoluteInput_row_center__activeLeft
+            },
+            {
+                location: Location.right,
+                className: this.props.managedClasses.absoluteInput_row_center__activeRight
+            }
+        ].forEach((locationsMappedToClassNames: ILocationsMappedToClassNames): void => {
+            classNames = this.props[locationsMappedToClassNames.location]
+                ? `${classNames} ${locationsMappedToClassNames.className}`
+                : classNames;
+        });
+
+        return classNames;
+    }
+
+    private handleOnChange = (e: React.ChangeEvent<HTMLSelectElement | HTMLInputElement>): void => {
+        const updatedProps: Partial<ICSSPositionProps> = this.assignUpdatedProps(
+            [this.positionKey, Location.top, Location.left, Location.right, Location.bottom],
+            e.target.dataset.location,
+            e.target.value
+        );
+
+        this.props.onChange(updatedProps);
+    }
+
+    private assignUpdatedProps(props: string[], updatedPropKey: string, updatedPropValue: string): Partial<ICSSPositionProps> {
+        let updatedProps: Partial<ICSSPositionProps> = {};
+
+        switch (updatedPropKey) {
+            case "position":
+                updatedProps[updatedPropKey] = updatedPropValue as PositionValue;
+                break;
+            case Location.left:
+            case Location.right:
+            case Location.top:
+            case Location.bottom:
+                updatedProps = this.getUpdatedPositions(props, updatedPropKey, updatedPropValue);
+                break;
+        }
+
+        return updatedProps;
+    }
+
+    private getUpdatedPositions(props: string[], updatedPropKey: Location, updatedPropValue: string): Partial<ICSSPositionProps> {
+        const updatedProps: Partial<ICSSPositionProps> = {};
+        const excludedProp: Location = this.getExcludedLocation(updatedPropKey);
+
+        props.forEach((prop: string): void => {
+            if (this.props[prop] && prop !== excludedProp) {
+                updatedProps[prop] = this.props[prop];
+            }
+        });
+
+        updatedProps[updatedPropKey] = parseInt(updatedPropValue, 10);
+
+        return updatedProps;
+    }
+
+    private getExcludedLocation(updatedPropKey: Location): Location {
```

Comment:
Do we need to add a simple comment above these functions to provide a little insight into their purpose? Obviously not full JSDoc syntax, but similar to our approach with all other packages?